phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less

```diff
@@ -440,6 +440,13 @@
     -fx-background-position: left;
 }
 
+.detailsPane {
+    .closeIcon {
+        -fx-border-width: 0;
+        -fx-border-style: none;
```

Comment:
You mean the `0` and `none`?
I'm not sure, I found this in the main.less file for the unity theme at another location....